files/www/lan/icon.svg

```diff
@@ -0,0 +1,20 @@
+<path d="m110.67 0.4893c-18.369 0-35.988 7.3744-48.96 20.473-12.972 13.099-20.249 30.852-20.249 49.352s7.2774 36.253 20.249 49.352c12.972 13.099 30.59 20.473 48.96 20.473 18.369 0 35.988-7.3744 48.96-20.473 12.972-13.099 20.249-30.852 20.249-49.352 0-18.5-7.2774-36.253-20.249-49.352-12.972-13.099-30.59-20.473-48.96-20.473zm0 12.663c14.983 0 29.35 6.0047 39.962 16.721 10.612 10.716 16.584 25.263 16.584 40.442s-5.9717 29.725-16.584 40.441c-10.612 10.716-24.979 16.721-39.962 16.721-14.983 0-29.35-6.0048-39.962-16.721-10.612-10.716-16.584-25.262-16.584-40.441s5.9717-29.726 16.584-40.442 24.979-16.721 39.962-16.721z" color="#000000" color-rendering="auto" fill="#de2c68" image-rendering="auto" shape-rendering="auto" solid-color="#000000" stroke="#000" stroke-width=".9786" style="isolation:auto;mix-blend-mode:normal;text-decoration-color:#000000;text-decoration-line:none;text-decoration-style:solid;text-indent:0;text-transform:none;white-space:normal"/>
+<polygon transform="matrix(-1.0238 0 0 1.0596 99.481 26.675)" points="81 41 73 41 64 50 73 59 81 59 75 53 88 53 88 47 75 47" fill="#fc3"/>
+<g transform="matrix(.0049944 0 0 -.0049944 75.436 92.18)" fill="#fff">
+<path d="m6525 9566c-167.54-119.19-121.86-736.67-81.799-557.38 3.3716-541.13 5.2062-1079.7-76.144-1613.7-55.537-723.1-97.828-1449.9-56.057-2174.9h-3296c-436.23 922.53-987.48 1800.2-1599 2608-337.18 446.78-547.99 998.48-890.92 1435.3-727.25 82.34 1072.1-2067.1 302.02-1092.7 379.5-685.79 696.04-1405.5 1079.2-2091.7 38.889-245.89 221.57-957.79 399.99-857.98-213.54-21.157-444.72 32.077-647.25-33.924-461.33-399.51-469.67-1061.1-655.98-1605-166.9-684.54-395.84-1361-538.02-2048.2-58.173-342.61 177.83-1264.9 87.125-907.33 214.76-465.88 1744.3-337.72 1322.4-244.25 3420.4 4.3345 6841.7-20.288 10261 13.755 343.27 314.85 278.6 1134.4 264 1007.5-25.186 455.53-218.54 890.16-314.44 1335.9-210.24 701.88-340.13 1433.3-600.5 2116.9-478.26 574.37-717 401.56-981.33 374.11-75.514 139.2 864.95 1569.3 516.39 1039.5 439.09 803.99 793.66 1646.5 1278.4 2425 256.41 161.86 249.01 795.1-90.84 462.39-339.99-550.28-619.44-1139.9-1034-1637.2-501.81-727.09-984.38-1473.4-1353.7-2278-977.54-42.058-1958.1-14.55-2937-22 0.371 1005.6-61.344 2012.7-122.61 3012.3-0.6991 419.9-0.2722 839.79-0.3906 1259.7-56.019 67.646-148.52 111.35-235 74zm-5824-9143c38.722-45.687-66.719 52.351 0 0z" fill="#fff"/>
+</g>
```

Comment:
muss das so?

ich glaub ja nicht!

Reply:
Router in weiß? Warum?

mal mit - mal ohne schwarzen rand?

nicht im bodensee ci?